scripts/default_config.py

```diff
@@ -132,9 +132,10 @@ def get_default_config():
     cfg.train.gamma = 0.1  # learning rate decay multiplier
     cfg.train.first_cycle_steps = 5
     cfg.train.cycle_mult = 1.
-    cfg.train.min_lr = 1e-5
-    cfg.train.max_lr = 0.1
+    cfg.train.min_lr = [1e-5]
```

Comment:
Why do you need a list here?

Reply:
These params are actually passed to CosineAnnealingCycleRestart only. Default setting would cause a runtime error if NBD is enabled with the mentioned scheduler. For other schedulers you generate min/max lrs right inplace taking into account the real number of parameter groups.